api/swagger/server/restapi/resource/swagger.yml

```diff
@@ -374,6 +374,50 @@ paths:
           description: Internal Server Error - The server has encountered a situation it does not know how to handle.
           schema:
             $ref: "#/definitions/Error"
+  /network:
+    get:
```

Comment:
There is already the endpoint massa/node that returns the current node.. Maybe it should be deprecated in favor of this one ?

Reply:
yeah we will deprecate the other , this returns all networks we can switch to + the current one